test/image/mocks/legend-constant-itemsizing.json

```diff
@@ -0,0 +1,169 @@
+{
+  "data": [
+    {
+      "mode": "markers",
+      "name": "markers",
+      "y": [ 1, 1, 1 ],
+      "marker": {
+        "size": [ 20, 40, 30 ],
+        "line": {
+          "width": 10,
+          "color": "#444"
+        }
+      }
+    },
+    {
+      "type": "bar",
+      "name": "bar",
+      "base": 2,
+      "y": [ 0.5, 0.5, 0.5 ],
+      "marker": {
+        "line": {
+          "width": 15
+        }
+      }
+    },
+    {
+      "mode": "lines",
+      "name": "lines",
+      "y": [ 3, 3, 3 ],
+      "line": {
+        "width": 30
+      }
+    },
+    {
+      "mode": "markers+lines+text",
+      "name": "all modes",
+      "y": [ 4, 4, 4 ],
+      "text": [ "a", "b", "c" ],
+      "textposition": "bottom left",
+      "line": {
+        "width": 20
+      },
+      "marker": {
+        "size": 50,
+        "line": {
+          "width": [ 10, 15, 14 ],
+          "color": "#444"
+        }
+      }
+    },
+    {
+      "type": "waterfall",
+      "name": "waterfall",
+      "base": 5,
+      "y": [ 1, -1, 1 ],
```

Comment:
Could you please add these lines into it?
<pre>
"y": [ 1, -0.5, null],
"measure": [ "", "", "total"],
</pre>

Reply:
done in https://github.com/plotly/plotly.js/pull/3732/commits/c6301a1b921a6b324acaea57b5853e5a550b5c7a